app/controllers/katello/api/v2/packages_controller.rb

```diff
@@ -12,10 +15,23 @@ def index
       super
     end
 
-    private
+    def auto_complete_name
+      rpms = Rpm.in_repositories(@repositories)
+      col = "#{Rpm.table_name}.name"
+      rpms = rpms.where("#{Rpm.table_name}.name ILIKE ?", "#{params[:term]}%").select(col).group(col).order(col).limit(20)
```

Comment:
It is also what scoped seach hard codes for its auto complete limits.  I'm not sure if page size is appropriate, nor that the user needs to change it.  I guess i can make it a constant somewhere.